### PR TITLE
Update whl_targets with overrides (follow up from#67)

### PR DESCRIPTION
**Pull request by fahhem**
This allows someone it still use `requirement("overridden-dep")`

Also, rename replacements to overrides in the code for better code search

### CODE REVIEW

**Comment by ali5h** [comment]
would you please check CI issue

**Comment by fahhem** [comment]
CI issue is that it cannot run black, not an issue with this PR
image

**Comment by ali5h** [comment]
would u please rebase on master? thanks

**Comment by fahhem** [comment]
okay, fixed the formatting and tests!

File: src/piptool.py [hunk 1]
```diff
@@ -189,7 +189,12 @@ def main():
     for req in reqs:
         name, version, extras = as_tuple(req)
         repo_name = repository_name(args.repo_prefix, name, version, python_version)
-        whl_targets["%s" % name] = "@%s//:pkg" % repo_name
+        if req in overrides:
```
Comment (ali5h) [inline src/piptool.py:192]: would you please avoid using `continue`

Comment (fahhem) [reply src/piptool.py:192]: what would you prefer?

a) two for loops, each over a subset of reqs (each subset created via a filter call?)
b) moving the lines 198-213 in 1 indentation and making the else clause longer?

Comment (ali5h) [reply src/piptool.py:192]: i think b is good enough

**Pull request merged**